code-scanning/42crunch.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
All the files related to this one need an ID of `crunch42`, in order to override the existing workflow. A subsequent rename to `42crunch` once the original has been removed would be great though!

Reply:
Good point about the rename waiting until we've removed the other 👍

Reply:
Done!